source/install/tarball.rst

```diff
@@ -6,13 +6,18 @@ Installing Percona Server for MongoDB from Binary Tarball
 
 You can find links to the binary tarballs under the *Generic Linux* menu item on the `Percona website`_
 
+There are two tarballs available:
+
+- ``percona-server-mongodb-4.4.0-1-x86_64.glibc2.17.tar.gz`` is the general tarball, compatible with any operation system but for Centos 6.
```

Comment:
Please set percona-server-mongodb-4.4.1-2 version, since percona-server-mongodb-4.4.0-1 was released in previous format, so example isn't valid